src/scheduler/random.rs

```diff
@@ -18,6 +18,33 @@ pub struct RandomScheduler {
     rng: Pcg64Mcg,
     iterations: usize,
     data_source: RandomDataSource,
+    current_seed: CurrentSeedDropGuard,
+}
+
+#[derive(Debug, Default)]
+struct CurrentSeedDropGuard {
+    inner: Option<u64>,
+}
+
+impl CurrentSeedDropGuard {
+    fn wipe(&mut self) {
+        self.inner = None
+    }
+
+    fn update(&mut self, seed: u64) {
+        self.inner = Some(seed)
+    }
+}
+
+impl Drop for CurrentSeedDropGuard {
+    fn drop(&mut self) {
+        if let Some(s) = self.inner {
+            eprintln!(
+                "failing seed:\n\"\n{}\n\"\npass the seed to `shuttle::replay_from_seed` to replay the failure.",
```

Comment:
If we decide to do replay based on environment variables, then the comment here needs to be updated.